HW3/task1.py

```diff
@@ -0,0 +1,11 @@
+import random
+
+N = int(input("Введите кол-во элементов массива: "))
+arr = [random.randint(1, N) for i in range(N)]
+print(*arr)
+x = int(input("Введите некоторое число Х="))
+count = 0
+for i in range(0, len(arr)):
+    if arr[i] == x:
```

Comment:
Функция `range` тут не нужна, можно напрямую проходиться по элементам списка.
```suggestion
for i in arr:
    if i == x:
```